src/modules/dashboard/components/Extensions/ExtensionDetails.tsx

```diff
@@ -373,9 +371,13 @@ const ExtensionDetails = ({
                       />
                     </div>
                   )}
-                  {extensionEnabled &&
+                  {extensionNotDeprecated &&
                     extension.enabledExtensionBody &&
-                    extension.enabledExtensionBody()}
+                    extension.initializationParams &&
+                    extension.enabledExtensionBody(
+                      extension.initializationParams,
+                      canInstall,
```

Comment:
So all of the concept of extensionBody which i introduced will be used when extension will be enabled, to support content in Extension (for our case upload addresses component, and addresses list) Here you are mixed things. Init params are used before we enable extension, so no need to touch this code at all